### PR TITLE
Return image bytes for text-to-image

### PR DESCRIPTION
**Pull request by osanseviero**

### CODE REVIEW

File: api-inference-community/api_inference_community/routes.py [hunk 1]
```diff
@@ -108,6 +109,18 @@ def emit(self, record):
...
+            buf = io.BytesIO()
+            outputs.save(buf, format="JPEG")
+            buf.seek(0)
+            img_bytes = buf.read()
+            b64_img_bytes = base64.b64encode(img_bytes)
+            return Response(
+                b64_img_bytes,
+                headers=headers,
+                status_code=200,
+                media_type="image/jpeg",
```
Comment (osanseviero) [inline api-inference-community/api_inference_community/routes.py:122]: Starlette uses `media_type` to automatically set the `content-type` header. Let me know if you would rather have `content-type` explicitly in `headers["content-type"]` for consistency.

Comment (Narsil) [reply api-inference-community/api_inference_community/routes.py:122]: Why are you still base64encoding ? You don't need to anymore.

For the header, it's fine. I don't really think it's worth it (as it's passed as-is anyway) but it's fine.

Comment (osanseviero) [reply api-inference-community/api_inference_community/routes.py:121]: Fixed

**Comment by osanseviero** [comment]
Done, I'm now returning the raw bytes instead of the encoded bytes. This should also make usage simpler (I was using `response.text` instead of `response.content`, which was causing the confusion).

**Review by Narsil — APPROVED** [review]
LGTM

**Pull request merged**